### PR TITLE
Footer nav text too small

### PR DESCRIPTION
**Pull request by mxsnyder**
I've embiggened the contraster, and made the "adjust contrast" text linked to operate the contraster.

### CODE REVIEW

File: dist/js/master.js [hunk 1]
```diff
@@ -1094,6 +1095,12 @@ bcpl.contraster = function ($, browserStorage) {
   */
 	var contrastButtonClickHandler = function contrastButtonClickHandler(clickEvent) {
 		var settings = clickEvent ? clickEvent.data : contrasterDefaults;
+		var $eventTarget = $(clickEvent.currentTarget);
+
+		if ($eventTarget.is(contrasterDefaults.selectors.toggleText)) {
+			$eventTarget.parent().find('input').trigger('click');
```
Comment (martypowell) [inline dist/js/master.js]: is it possible to use closest here?

File: js/contraster.js [hunk 2]
```diff
@@ -21,6 +22,12 @@ bcpl.contraster = (($, browserStorage) => {
 	 */
 	const contrastButtonClickHandler = (clickEvent) => {
 		const settings = clickEvent ? clickEvent.data : contrasterDefaults;
+		const $eventTarget = $(clickEvent.currentTarget);
+
+		if ($eventTarget.is(contrasterDefaults.selectors.toggleText)) {
+			$eventTarget.parent().find('input').trigger('click');
```
Comment (martypowell) [inline js/contraster.js]: i mean is it possible to use closest here?

Comment (mxsnyder) [reply js/contraster.js]: Technically yes, but I just need the immediate parent.

**Comment by ascire** [comment]
> embiggened

A perfectly cromulent word.

**Pull request merged**